.github/workflows/docs.yml

```diff
@@ -0,0 +1,44 @@
+name: "Publish Docs"
+
+on:
+  push:
+    branches: [master]
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: pnpm/action-setup@v3
+        with:
+          version: 8
+      - name: Use Node.js ${{ matrix.node-version }}
```

Comment:
wait a sec, matrix isn't declared for this workflow?
Does it fall back to defaults?